### DISCUSSION TITLE
How can I import a Pinia store globally so I don't have to import it in every single component?

### DISCUSSION THREAD

**Discussion by wouter-muller** [vuejs/pinia#2309] [Help and Questions] [answered]
I love Vue and Pinia, but there is one thing that is driving me insane.

I have some Pinia stores that I end up importing in almost all of my components. This gets very tedious, even with autocomplete. I would love a way that I can say "hey, this store should be globally accessible" so I don't need to import it every time manually. Does anyone know of a way how to do that?

I am using the composition API...

**Comment by posva** [comment, marked as answer]
Use auto imports: https://github.com/antfu/unplugin-auto-import

[Nuxt](https://nuxt.com/) has it by default

**Comment by wouter-muller** [comment]
Nice! I'll give that a try but it sounds like exactly what I need... Thank you so much!
